docs/configuration-reference.md

```diff
@@ -282,7 +283,7 @@ lang: <ISO 639-1 code -  example: 'en'>
 
 Configure the REST API in opsdroid.
 
-By default, opsdroid will start a web server accessible only to localhost on port `8080` (or `8443` if ssl details are provided). For more information see the [REST API docs](rest-api).
+By default, opsdroid will start a web server accessible only to localhost on port `8080` (or `8443` if ssl details are provided). For more information see the [REST API docs](https://github.com/opsdroid/opsdroid/blob/master/docs/rest-api.md).
```

Comment:
I have testing this link on my end and it works fine how it was, could you revert it?